verto/tests/RelativeLinkTest.py

```diff
@@ -43,6 +43,36 @@ def test_long_path(self):
         expected_string = self.read_test_file(self.processor_name, 'long_path_expected.html', strip=True).strip()
         self.assertEqual(expected_string, converted_test_string)
 
+    def test_query_parameter(self):
+        '''Tests that paths with query parameter work.
+        '''
+        test_string = self.read_test_file(self.processor_name, 'query_parameter.md')
+        processor = RelativeLinkPattern(self.ext, self.md.parser)
+        self.assertIsNotNone(re.search(processor.compiled_re, test_string))
+        converted_test_string = markdown.markdown(test_string, extensions=[self.verto_extension])
+        expected_string = self.read_test_file(self.processor_name, 'query_parameter_expected.html', strip=True).strip()
+        self.assertEqual(expected_string, converted_test_string)
+
+    def test_multiple_query_parameters(self):
+        '''Tests that paths with multiple query parameters work.
+        '''
+        test_string = self.read_test_file(self.processor_name, 'multiple_query_parameters.md')
+        processor = RelativeLinkPattern(self.ext, self.md.parser)
+        self.assertIsNotNone(re.search(processor.compiled_re, test_string))
+        converted_test_string = markdown.markdown(test_string, extensions=[self.verto_extension])
+        expected_string = self.read_test_file(self.processor_name, 'multiple_query_parameters_expected.html', strip=True).strip()
+        self.assertEqual(expected_string, converted_test_string)
+
+    def test_trailing_question_mark(self):
+        '''Tests that paths with trailing question marks.
+        '''
+        test_string = self.read_test_file(self.processor_name, 'trailing_question_mark.md')
+        processor = RelativeLinkPattern(self.ext, self.md.parser)
+        self.assertIsNotNone(re.search(processor.compiled_re, test_string))
+        converted_test_string = markdown.markdown(test_string, extensions=[self.verto_extension])
+        expected_string = self.read_test_file(self.processor_name, 'trailing_question_mark_expected.html', strip=True).strip()
+        self.assertEqual(expected_string, converted_test_string)
```

Comment:
please adding spacing in your test methods, see the existing methods for how we usually do this